docs/modules/publisher-migrationpath.md

```diff
@@ -55,6 +55,78 @@ Please consider migrating your old published nodes JSON files that use `NodeId`
+| --pf, --publishfile=VALUE               |  yes                     |                 |
+| --tc, --telemetryconfigfile=VALUE       |  no                      |                 |
+| --s, --site=VALUE                       |  yes                     |                 |
+| --ic, --iotcentral                      |  no                      |                 |
+| --sw, --sessionconnectwait=VALUE        |  no                      |                 |
+| --mq, --monitoreditemqueuecapacity=VALUE|  yes                     |                 |
+| --di, --diagnosticsinterval=VALUE       |  yes                     |                 |
+| --ns, --noshutdown=VALUE                |  no                      |                 |
+| --rf, --runforever                      |  no                      |                 |
+| --lf, --logfile=VALUE                   |  yes                     |                 |
+| --lt, --logflushtimespan=VALUE          |  yes                     |                 |
+| --ll, --loglevel=VALUE                  |  yes                     |                 |
+| --ih, --iothubprotocol=VALUE            |  yes                     |                 |
+| --ms, --iothubmessagesize=VALUE         |  yes                     |                 |
+| --si, --iothubsendinterval=VALUE        |  yes                     |                 |
+| --dc, --deviceconnectionstring=VALUE    |  yes                     |                 |
+| --c, --connectionstring=VALUE           |  no                      |  use --dc, --deviceconnectionstring=VALUE               |
+| --hb, --heartbeatinterval=VALUE         |  yes                     |                 |
+| --sf, --skipfirstevent=VALUE            |  no                      |                 |
+| --pn, --portnum=VALUE                   |  no                      |                 |
+| --pa, --path=VALUE                      |  no                      |                 |
+| --lr, --ldsreginterval=VALUE            |  no                      |                 |
+| --ol, --opcmaxstringlen=VALUE           |  yes                     |                 |
+| --ot, --operationtimeout=VALUE          |  yes                     |                 |
+| --oi, --opcsamplinginterval=VALUE       |  yes                     |                 |
+| --op, --opcpublishinginterval=VALUE     |  yes                     |                 |
+| --ct, --createsessiontimeout=VALUE      |  yes                     |                 |
+| --ki, --keepaliveinterval=VALUE         |  yes                     |                 |
+| --kt, --keepalivethreshold=VALUE        |  yes                     |                 |
+| --aa, --autoaccept                      |  yes                     |                 |
+| --tm, --trustmyself=VALUE               |  yes                     |                 |
+| --to, --trustowncert                    |  no                      | same as trustmyself                |
```

Comment:
Can you please add `--` in front of command line options mentioned in `Alternative` column ?

Reply:
added